src/InstrumentationEngine.Packages.sln

```diff
@@ -64,19 +64,18 @@ Global
 		{0CA8A56A-B19D-4B69-82A2-3DCBD95F8A05}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{0CA8A56A-B19D-4B69-82A2-3DCBD95F8A05}.Release|Any CPU.Build.0 = Release|Any CPU
 		{0CA8A56A-B19D-4B69-82A2-3DCBD95F8A05}.Release|ARM64.ActiveCfg = Release|Any CPU
-		{0CA8A56A-B19D-4B69-82A2-3DCBD95F8A05}.Release|ARM64.Build.0 = Release|Any CPU
 		{0CA8A56A-B19D-4B69-82A2-3DCBD95F8A05}.Release|x64.ActiveCfg = Release|Any CPU
 		{0CA8A56A-B19D-4B69-82A2-3DCBD95F8A05}.Release|x86.ActiveCfg = Release|Any CPU
 		{3D3BA42C-4A1E-4135-B750-4D8CB0A7117F}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{3D3BA42C-4A1E-4135-B750-4D8CB0A7117F}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{3D3BA42C-4A1E-4135-B750-4D8CB0A7117F}.Debug|ARM64.ActiveCfg = Debug|arm64
-		{3D3BA42C-4A1E-4135-B750-4D8CB0A7117F}.Debug|ARM64.Build.0 = Debug|arm64
+		{3D3BA42C-4A1E-4135-B750-4D8CB0A7117F}.Debug|ARM64.ActiveCfg = Debug|ARM64
+		{3D3BA42C-4A1E-4135-B750-4D8CB0A7117F}.Debug|ARM64.Build.0 = Debug|ARM64
 		{3D3BA42C-4A1E-4135-B750-4D8CB0A7117F}.Debug|x64.ActiveCfg = Debug|x64
 		{3D3BA42C-4A1E-4135-B750-4D8CB0A7117F}.Debug|x86.ActiveCfg = Debug|x86
 		{3D3BA42C-4A1E-4135-B750-4D8CB0A7117F}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{3D3BA42C-4A1E-4135-B750-4D8CB0A7117F}.Release|Any CPU.Build.0 = Release|Any CPU
-		{3D3BA42C-4A1E-4135-B750-4D8CB0A7117F}.Release|ARM64.ActiveCfg = Release|Any CPU
-		{3D3BA42C-4A1E-4135-B750-4D8CB0A7117F}.Release|ARM64.Build.0 = Release|Any CPU
```

Comment:
VS tends to change these solution entries automatically and not always to the correct value. Was the conversion from Any CPU to ARM64 intended here?

Reply:
This change is intended; we are building the Headers package differently for AnyCPU (which contains x86 and x64 folders) and ARM64 (which contains an ARM64 folder) since we will publish the AnyCPU Headers package publicly to NuGet.org while keeping the ARM64 headers package internal since we are not ready to announce that CLRIE officially supports ARM64 yet.